OpenRA.Mods.Common/Traits/CashTrickler.cs

```diff
@@ -62,6 +64,7 @@ public CashTrickler(CashTricklerInfo info)
 		protected override void Created(Actor self)
 		{
 			resources = self.Owner.PlayerActor.Trait<PlayerResources>();
+			cloaks = self.TraitsImplementing<Cloak>().ToArray();
```

Comment:
I meant that `ModifyCash` should be overrideable so you can drop this trade lookups. Thirdparty mods can then inherit and create their own `CloakedCashTrickler`.

Reply:
I'm sure this is gonna be an issue in upstream in TS